index.html

```diff
@@ -62,12 +59,11 @@
           </div>
           <div class="date-info">2022.04.14(MON) ~ (TUE)</div>
           <div class="location">
-            <div class="locate fs-5">@Nations of the World, Allow your citizens to learn Web Development!</div>
+            <div class="locate fs-5">@Starting this date, come and join to learn Web Development!</div>
           </div>
         </section>
       </section>
 
-      <!-- program -->
       <section id="program" class="program-bg">
         <div class="container-md p-0 my-0 mx-auto">
           <h3 class="cocogoose text-white text-center h4 fw-bold pt-5">Main Program</h3>
```

Comment:
- [ ] When I see the page on the mobile screen the design is appearing differently for the cards, Please make sure it matches the design on the template for all screens. You may want to reduce the font size.

### Your design

![image](https://user-images.githubusercontent.com/83803414/158641348-7cef5544-ad4e-4a22-bc40-24e35379d55e.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/158641397-23e92e6c-fc30-4352-941a-3be115eeb5b5.png)